spread.yaml

```diff
@@ -373,7 +381,7 @@ suites:
 
     tests/regression/:
         summary: Regression tests for snapd
-        systems: [-fedora-*]
+        systems: [-fedora-*, -opensuse-*]
```

Comment:
Can you please document why those are disabled? Ideally we'd run them and skip the parts that involve confinement.